### PR TITLE
Add TXT record to uwunotislove.is-a.dev

### PR DESCRIPTION
**Pull request by Moodygd761**
i already have a domain i just want to verify it with GitHub pages

### CODE REVIEW

File: domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "uwunotislove",
+        "email": "moody761@hotmail.com"
+    },
+    "record": {
+        "TXT": "0964074425"
```
Comment (CuteDog5695) [inline domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json:7]:
```suggestion
        "TXT": ["0964074425"]
```
Need to update docs ig

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json (hunk 1 above)
Comment (Moodygd761) [reply domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json]: i updated

**Review by iostpa — APPROVED** [review]
Should be fine now

File: domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json
Comment (CuteDog5695) [inline domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json:1]: So im actually dumb, please remove `.is-a.dev` from the name

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json
Comment (Moodygd761) [reply domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json:1]: i removed .is-a.dev from the name

**Review by iostpa — APPROVED** [review]
Now its actually fine

**Pull request merged**